From klaas.van.gend@mvista.com  Tue Oct 27 10:34:35 2009
From: Klaas van Gend <klaas.van.gend@mvista.com>
Date: Thu, 22 Oct 2009 07:50:59 +0200
Subject: staging: comedi: multiq3: remove warning on braces
To: gregkh@suse.de
Cc: Klaas van Gend <klaas.van.gend@mvista.com>
Message-ID: <1256190660-14913-1-git-send-email-klaas.van.gend@mvista.com>


Remove braces in if statements to make the file exhibit less warnings
when checked using checkpatch.pl.

Signed-off-by: Klaas van Gend <klaas.van.gend@mvista.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/multiq3.c |   14 +++++---------
 1 file changed, 5 insertions(+), 9 deletions(-)

--- a/drivers/staging/comedi/drivers/multiq3.c
+++ b/drivers/staging/comedi/drivers/multiq3.c
@@ -144,9 +144,8 @@ static int multiq3_ao_insn_read(struct c
 	int i;
 	int chan = CR_CHAN(insn->chanspec);
 
-	for (i = 0; i < insn->n; i++) {
+	for (i = 0; i < insn->n; i++)
 		data[i] = devpriv->ao_readback[chan];
-	}
 
 	return i;
 }
@@ -262,11 +261,10 @@ static int multiq3_attach(struct comedi_
 	dev->iobase = iobase;
 
 	irq = it->options[1];
-	if (irq) {
+	if (irq)
 		printk("comedi%d: irq = %u ignored\n", dev->minor, irq);
-	} else {
+	else
 		printk("comedi%d: no irq\n", dev->minor);
-	}
 	dev->board_name = "multiq3";
 	result = alloc_subdevices(dev, 5);
 	if (result < 0)
@@ -332,12 +330,10 @@ static int multiq3_detach(struct comedi_
 {
 	printk("comedi%d: multiq3: remove\n", dev->minor);
 
-	if (dev->iobase) {
+	if (dev->iobase)
 		release_region(dev->iobase, MULTIQ3_SIZE);
-	}
-	if (dev->irq) {
+	if (dev->irq)
 		free_irq(dev->irq, dev);
-	}
 
 	return 0;
 }
